### PR TITLE
[GH-13461] add unit tests to the RemoveChannelUsersCmd mmctl command

### PR DESCRIPTION
**Pull request by fedealconada**
#### Summary
Adds unit tests to the RemoveChannelUsersCmd mmctl command

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/13461

### CODE REVIEW

File: cmd/mattermost/commands/channel_test.go [hunk 1]
```diff
@@ -30,24 +29,53 @@ func TestJoinChannel(t *testing.T) {
...
 func TestRemoveChannel(t *testing.T) {
+
 	th := Setup(t).InitBasic()
 	defer th.TearDown()
 
 	channel := th.CreatePublicChannel()
 
-	th.CheckCommand(t, "channel", "add", th.BasicTeam.Name+":"+channel.Name, th.BasicUser2.Email)
-
-	// should fail because channel does not exist
```
Comment (mgdelacroix) [inline cmd/mattermost/commands/channel_test.go:40]: This case is missing after the refactor

Comment (fedealconada) [reply cmd/mattermost/commands/channel_test.go:40]: I've never seen this! Sorry about that @mgdelacroix I'll update this ASAP @jasonblais

Comment (fedealconada) [reply cmd/mattermost/commands/channel_test.go:40]: Done @mgdelacroix @jasonblais

Comment (mgdelacroix) [reply cmd/mattermost/commands/channel_test.go:40]: No worries! Thanks a lot for updating the PR :)

**Pull request merged**